pkg/validate/dynamic/dynamic.go

```diff
@@ -597,7 +612,7 @@ func (dv *dynamic) validateCIDRRanges(ctx context.Context, subnets []Subnet, add
 func (dv *dynamic) validateVnetLocation(ctx context.Context, vnetr azure.Resource, location string) error {
 	dv.log.Print("validateVnetLocation")
 
-	vnet, err := dv.virtualNetworks.Get(ctx, vnetr.ResourceGroup, vnetr.ResourceName, "")
+	vnet, err := dv.virtualNetworks.Get(ctx, vnetr.ResourceGroup, vnetr.ResourceName, nil)
```

Comment:
I think we should make it explicit that we're deliberately not providing options here... e.g.:

```
options := nil
vnet, err := dv.virtualNetworks.Get(ctx, vnetr.ResourceGroup, vnetr.ResourceName, options)
```

I think that these options may end up being relevant in the future

Reply:
For a few reasons, I'd like to leave it as it is:
1. I think the current way is better, because in terms of explicitness, there's no difference IMHO, and the current way is simpler than the suggested way.
2. I've done some migrations and most of them didn't use this argument. If we change them, it will make a lot of redundant lines.
3. This PR aims to migrate from old SDK to new SDK. Since the old one didn't do in that way, changing the way is out of scope.